install-tools.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+# -*- coding: utf-8 -*-
+## Convenience script for installing all the tools at once
+
+REPO='https://pkg.caveoforig.in/fhtk/';
```

Comment:
Would be better to have the repo point back to pret on github, or to some host maintained by pret keyholders.